From b682fd6d7e582746672a97146f66aa8120000371 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Wed, 23 Apr 2014 12:21:37 +0300
Subject: [PATCH] CMakeLists: don't hard code thumb code generation

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bd29984..c4f019d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,7 +87,7 @@ if(ANDROID_PLATFORM)
       add_subdirectory(android/NE10Demo/jni)
     endif()
 elseif(GNULINUX_PLATFORM)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mthumb-interwork -mthumb -march=armv7-a -mfpu=vfp3")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mthumb-interwork -march=armv7-a -mfpu=vfp3")
     set(CMAKE_ASM_FLAGS "${CMAKE_C_FLAGS} -mthumb-interwork -mthumb -march=armv7-a -mfpu=neon")
 elseif(IOS_PLATFORM)
     #set minimal target ios version.If not provided this option, Xcode
-- 
2.1.1

